imod/mf6/out/__init__.py

```diff
@@ -227,6 +227,7 @@ def open_cbc(
     flowja: bool = False,
     simulation_start_time: Optional[np.datetime64] = None,
     time_unit: Optional[str] = "d",
+    uzf: bool = False,
```

Comment:
I did a quick search across the mf6 IO docs, there are multiple packages with their own budget file.

In this case, I think I'd prefer having a specific function to deal with the cbc file instead? `open_cbc_uzf` or something? Because we might need to add additional methods for the water mover, the lake package, etc; and we don't want to have five boolean args.

Reply:
I removed this optional argument. I changed the output budget keys to a more general form that should work for all budget files. To identify the advanced packages, I check the header keys for the label 'gwf'. As far as I could see, that's unique to the advance packages.